dbms/src/Storages/Page/V3/tests/gtest_page_storage.cpp

```diff
@@ -1495,7 +1495,7 @@ try
         batch.putExternal(1999, 0);
         batch.putExternal(2000, 0);
         page_storage->write(std::move(batch));
-        ASSERT_EQ(page_storage->getMaxId(), 2000);
+        ASSERT_EQ(page_storage->getMaxId(), 0);
```

Comment:
why this is 0?

Reply:
`max_page_id` is only used after restart. And I don't want to add many `hasPrefix` check at runtime. So the runtime calculation is removed now.